app/forms/recaptcha_enterprise_form.rb

```diff
@@ -24,6 +24,12 @@ def recaptcha_result
           token: recaptcha_token,
           siteKey: IdentityConfig.store.recaptcha_site_key,
           expectedAction: recaptcha_action,
+          userInfo: {
+            accountId: user&.uuid,
+            userInfo: {
+              email: encrypted_email
```

Comment:
Wanted to leverage the already existing encrypted version of email since I believe that persists, but It looks like at the time of this we won't have the encryption for phone yet since we dont create a phone configuration until we confirm the phone. Would we mind having a temporary encryption for phone using parsed_phone in the PhoneRecaptchaForm?

Reply:
I'm not sure we want to send the encrypted email.

Reply:
+1 to Mitchell, `encrypted_email` is our symmetrically encrypted copy of the email, are we looking for a stable identifier for the email like a hash?

Reply:
Yes, Would there be one we want or should I just hash it?

Reply:
I'm not sure we'd want to use our internal hash either. This probably requires some conversation with Security, we may need to do something like have a separate hash that isn't used for anything but this?

Reply:
Would Email fingerprint be something to consider? I know we allow that to be used for logging purposes already.

Reply:
Yup, I will talk to security. Might be worth to just make a separate hash for now.